integration_tests/integration/ageOfPrisoner.spec.ts

```diff
@@ -31,7 +31,6 @@ context('Age of the prisoner', () => {
               locationId: 234,
             },
             startedByUserId: 'TEST_GEN',
```

Comment:
don't need these here as this page is where that data is set

Reply:
yep!